### DISCUSSION TITLE
Definition of fburst

### DISCUSSION THREAD

**Discussion by ceband** [mboquien/cigale#94] [Q&A] [answered]
I am writing to clarify what exactly fburst parameter means in SFHdelayed. In the manual it says that it is the mass fraction of the burst, and in your 2019 paper I saw the definition "fraction of stars formed in the second burst relative to the total mass of stars ever formed". So am I right to assume that this is the mass fraction of the burst at the moment of the burst's onset with respect to the older population? And then if it is high then the mass of the galaxy would be primarily dictated by the burst and not the older component, by definition? 
Thanks in advance

**Comment by mboquien** [comment, marked as answer]
This is the mass formed in the burst divided by the mass formed in the burst plus the mass formed in the main component. I think your conclusion is right. I think in most cases the actual burst fraction should not be very large.
